src/components/navbar/index.cjsx

```diff
@@ -103,7 +103,7 @@ module.exports = React.createClass
               className="signout"
               eventKey={4}
               key='dropdown-item-logout'>
-                <SignOut className='btn btn-link btn-xs'>Sign Out</SignOut>
+                <SignOut className='btn btn-link btn-xs'>Log Out</SignOut>
```

Comment:
Ah, the name of the component should probably change too (so it's easier to find)